client/src/Components/UserSignUp.js

```diff
@@ -34,13 +35,61 @@ function SignUp() {
   }));
   const classes = useStyles()
 
+  const SignUp = props => {
```

Comment:
Two options: 
You delete the `const SignUp = props => {` line here, then pass `props` to the top function declaration of Sign up or you delete the `function SignUp() {` line up top  on line 15. Either will work. Just delete the line, not also a `}` at the bottom. We have the correct number down there.